tests/resources/model_compatibility/check_backwards_compatibility.sh

```diff
@@ -0,0 +1,51 @@
+# CONSTANTs/ shouldn't change
+REGRESSION_MODULE="medcat.utils.regression.regression_checker"
+REGRESSION_OPTIONS="--strictness STRICTEST --require-fully-correct"
+
+# CHANGABLES
+# target models
+DL_LINK="https://cogstack-medcat-example-models.s3.eu-west-2.amazonaws.com/medcat-example-models/all_fake_medcat_models.zip"
+ZIP_FILE_NAME="all_fake_medcat_models.zip"
+# target regression set
+REGRESSION_TEST_SET="tests/resources/regression/testing/test_model_regresssion.yml"
+# folder to house models under test
+MODEL_FOLDER="fake_models"
+
+# START WORK
+
+echo "Downloading models"
+wget $DL_LINK
+# Create folder if it doesn't exit
+if [ ! -d "$MODEL_FOLDER" ]; then
+    mkdir "$MODEL_FOLDER"
+    CREATED=1
```

Comment:
pretty defensive? would it matter if this dir is removed between runs? Maybe just do:
`
mkdir -p $MODEL_FOLDER
unzip $ZIP_FILE_NAME -d $MODEL_FOLDER
...
rm -r  $MODEL_FOLDER
`

Reply:
The idea was that if the model existed before, then it was "meant to be there" and may have things that are expected in there. And I didn't want to remove other stuff that's (potentially) in there at the end.

With that said, you're probably right - there shouldn't really be a reason for that to be the case.